src/components/parameters.tsx

```diff
@@ -69,12 +81,25 @@ export function useParameterState(paramName) {
     return [paramLocalState, handleChangeParamLocalState];
 }
 
-const Parameters = ({ showParameters, hideParameters }) => {
+const Parameters: FunctionComponent<
+    PropsWithChildren<{
+        showParameters: boolean;
+        hideParameters: (
+            event?:
+                | Parameters<NonNullable<ModalProps['onClose']>>[0]
+                | Parameters<NonNullable<ButtonProps['onClick']>>[0],
+            reason?: Parameters<NonNullable<ModalProps['onClose']>>[1]
+        ) => void;
+    }>
+> = (props) => {
     const [tabIndex, setTabIndex] = useState(0);
 
-    function TabPanel(props) {
+    function TabPanel<T = number>(
+        props: PropsWithChildren<
+            TypographyTypeMap<{ index: T; value: T }>['props']
+        >
+    ): ReactElement {
```

Comment:
that is to keep the initial capacity of passing any type of data if wanted and because there is possibly others properties passed.  
`TabPanel` is just an extra layer around `Typography` component.